src/templates/static/legal/tac.jsx

```diff
@@ -1,139 +1,15 @@
-                        ],
-                    },
-                ]}
-            />
-        </div>
-
-        <div className='gr-9 gr-12-p gr-12-m gr-padding-10'>
-            <SidebarContentContainer>
-                <SidebarContent id='legal-binary'>
-                    <TermsAndConditionsBinary />
-                </SidebarContent>
-
-                <SidebarContent id='affiliate'>
-                    <Affiliate />
-                </SidebarContent>
-
-                <SidebarContent id='code-of-conduct'>
-                    <CodeOfConduct />
-                </SidebarContent>
-
-                <SidebarContent id='legal-mt'>
-                    <TermsAndConditionsMT />
-                </SidebarContent>
-
-                <SidebarContent id='datafeed'>
-                    <Datafeed />
-                </SidebarContent>
-
-                <SidebarContent id='privacy'>
-                    <Privacy />
-                </SidebarContent>
-
-                <SidebarContent id='conflicts-policy'>
-                    <ConflictsPolicy />
-                </SidebarContent>
-
-                <SidebarContent id='copyright'>
-                    <Copyright />
-                </SidebarContent>
-
-                <SidebarContent id='complaints'>
-                    <Complaints />
-                </SidebarContent>
-
-                <SidebarContent id='risk-binary'>
-                    <RiskDisclaimerBinary />
-                </SidebarContent>
-
-                <SidebarContent id='risk-mt'>
-                    <RiskDisclaimerMT />
-                </SidebarContent>
-
-                <SidebarContent id='order-execution-binary'>
-                    <OrderExecutionBinary />
-                </SidebarContent>
-
-                <SidebarContent id='order-execution-mt'>
-                    <OrderExecutionMT />
-                </SidebarContent>
-
-                <SidebarContent id='payment-agents'>
-                    <PaymentAgents />
-                </SidebarContent>
-
-                <SidebarContent id='api'>
-                    <Api />
-                </SidebarContent>
-
-                <SidebarContent id='introducing-brokers'>
-                    <IntroducingBrokers />
-                </SidebarContent>
-            </SidebarContentContainer>
+    <div className='tnc center-text'>
+        <img src={it.url_for('images/pages/terms-and-conditions/binary-to-deriv.svg')} alt={it.L('Binary to Deriv')} className='tnc_img' />
+        <h2>{it.L('We have updated our terms and conditions')}</h2>
+        <div className='tnc_text'>
+            <p>{it.L('We have updated our terms of service as part of our rebranding from [_1] to Deriv. You can review the updated terms and conditions at Deriv.com. Note that all [_1] users are now bound by the terms and conditions of Deriv. ', it.website_name)}</p>
         </div>
+        <a className='button tnc_btn' href='https://deriv.com/terms-and-conditions' target='_blank' rel='noopener noreferrer'>
```

Comment:
should we have a js function to redirect this to different languages based on the page language?

Reply:
This will do the job:
`it.language.toLowerCase()`